src/mumble/AudioOutputUser.h

```diff
@@ -26,9 +26,10 @@ class AudioOutputUser : public QObject {
 	AudioOutputUser(const QString &name);
 	~AudioOutputUser() Q_DECL_OVERRIDE;
 	const QString qsName;
-	float *pfBuffer = nullptr;
-	float *pfVolume = nullptr;
-	float fPos[3]   = { 0.0, 0.0, 0.0 };
+	float *pfBuffer        = nullptr;
+	float *pfVolume        = nullptr;
+	unsigned int *piOffset = nullptr;
```

Comment:
This should be a smart-pointer unless there is a good reason against it?
```suggestion
	std::unique_ptr<unsigned int []> piOffset;
```

Reply:
I would also use `uint8_t` instead of `unsigned int`.

Reply:
Why? It doesn't seem that this has to be guaranteed to be an 8-bit wide number but this is likely to cause performance because chances are the processor needs extra instructions to deal with this fixed-width type